ppr-ui/src/views/amendment/AmendRegistration.vue

```diff
@@ -0,0 +1,261 @@
+<template>
+  <v-container v-if="dataLoaded" class="view-container pa-15 pt-14" fluid style="min-width: 960px;">
+    <div class="container pa-0" style="min-width: 960px;">
+      <v-row no-gutters>
+        <v-col cols="9">
+          <h1>Registration Amendment</h1>
+          <div style="padding-top: 25px; max-width: 875px;">
+            <p class="ma-0">
+              Review the current information for this registration as of
+              <b>{{ asOfDateTime }}.</b><br/>
+            </p>
+          </div>
+          <caution-box class="mt-9" :setMsg="cautionTxt"/>
+          <registration-length-trust class="mt-15" :isSummary="true" :defaultRegistrationType="registrationType" />
+          <div class="summary-header mt-15 pa-4 rounded-top">
+            <v-icon color="darkBlue">mdi-account-multiple-plus</v-icon>
+            <label class="pl-3">
+              <strong>Registering Party, Secured Parties, and Debtors</strong>
+            </label>
+          </div>
+          <div style="padding-top: 25px; max-width: 875px;">
+            <p class="ma-0">
+              The Registering Party has been added based on your account information and cannot be changed here.
+            </p>
+          </div>
+          <h3 class="pt-6 px-1">Original Registering Party</h3>
+          <registering-party-summary class="pt-4" :setEnableNoDataAction="false" />
+          <h3 class="pt-6 px-1">Secured Parties</h3>
+          <secured-parties class="pt-4" />
+          <h3 class="pt-6 px-1">Debtors</h3>
+          <debtors />
+          <collateral class="mt-15" />
```

Comment:
Hey steph can you add ```:setRegistrationType="registrationType"``` here? It will be needed once my PR is merged

Reply:
done